Give-Hope-org/give-hope-fn
# changes regarding the login page

**What does this PR do?**

This PR does the following:

- Adds a header navbar to the login form to give the user access to routes to other pages.
- Fixes the fixed navbar bug that was hiding the login form.
- Changes the login illustration to a more relevant one.
- Hides the illustration on mobile and tablet.

**Description of Task to be Completed?**    

 the PR includes screenshots of the changes made on large screens, landscape phones, and tablets. These screenshots provide visual confirmation of the changes made and help the reviewer to understand the impact of the changes on different devices.      

**How should this be manually tested?**   
 
- navigate to the login route    
- check the responsiveness  on different screens (mobile, laptop, tablet, large screens)     

**Any background context you want to provide?**

None   

**What are the relevant pivotal tracker/Trello stories?**
  
stories related to the authentication UI, especially the login component on the client side     

**Screenshots:**

### Large Screens
![Large screens](https://www.linkpicture.com/q/login_44.png)

### Landscape Phones
![landscape phones](https://www.linkpicture.com/q/landscape_2.png)

### Tablet
![Tablet](https://www.linkpicture.com/q/tablet_1.png)

### Phone     
![Phone](https://www.linkpicture.com/q/phone_3.png)
